binary/script/embed.ipxe

```diff
@@ -6,4 +6,21 @@ echo Welcome to Neverland!
 # Allow the operator to drop to a shell
 prompt --key 0x02 --timeout 2000 Press Ctrl-B for the iPXE command line... && shell ||
 
+set vlan-id ${43.116:string}
```

Comment:
I'm confused. If this is the script embedded in our ipxe binaries, at this point I would have expected that DHCP hasn't happened yet from the perspective of ipxe... How would that value have been populated at this point?

Do we need to split out an explicit dhcp from the autoboot part?

Reply:
That's a great point. I will add a comment above this value to explain. My understanding of iPXE is that "the original vendor PXE DHCP request [is] available within [the chainloaded] UEFI iPXE [and] BIOS iPXE" -- [ref](https://github.com/ipxe/ipxe/issues/188)

Reply:
Thank you for adding the comment in there!